var/spack/repos/builtin/packages/hiop/package.py

```diff
@@ -100,10 +103,6 @@ class Hiop(CMakePackage, CudaPackage, ROCmPackage):
         description="Enable/disable cuSovler LU refactorization",
     )
 
-    depends_on("c", type="build")  # generated
-    depends_on("cxx", type="build")  # generated
-    depends_on("fortran", type="build")  # generated
```

Comment:
Does the package depend on any of these compilers? Or have you confirmed it does not depend on any of them?

Reply:
This part is redundant, as you can find these three lines around [L78-L80](https://github.com/spack/spack/blob/fc7f9f056499ac6702c6db936da8776c4e4373f8/var/spack/repos/builtin/packages/hiop/package.py#L78-L80)
Or maybe I should keep this three lines and remove the lines on L78-L80, to keep the `depends_on` in the same section.

Reply:
🤦‍♀️ I neglected to open the file up and check. Thanks for letting me know.

Reply:
Please move lines 78-80 so they are with the other dependency directives.

Reply:
done!

Reply:
Thanks.